src/form.js

```diff
@@ -313,45 +307,19 @@ define([
             return writer.createXForm(this);
         },
         /**
-         * Goes through and grabs all of the data nodes (i.e. nodes that are
-         * only data nodes, possibly with a bind) without any kind of control.
-         * Returns a flat list of these nodes (list items are mugs).
-         */
-        getDataNodeList: function () {
-            return this.dataTree.treeMap(function(node){
-                if (!node.getValue() || node.isRootNode) {
-                    return null;
-                }
-                var mug = node.getValue();
-
-                if (!mug.options.isDataOnly) {
-                    return null;
-                } else {
-                    return mug;
-                }
-            });
-        },
-        /**
-         * Walks through both internal trees (data and control) and grabs
+         * Walks through internal tree and grabs
          * all mugs that are not (1)Choices.  Returns
          * a flat list of unique mugs.  This list is primarily for the
          * autocomplete skip logic wizard.
          */
         getMugList: function () {
-            var treeFunc, cList, dList;
-
-            treeFunc = function (node) {
+            return this.controlTree.treeMap( function (node) {
                 if(node.isRootNode) {
                     return;
                 }
 
                 return node.getValue();
-            };
-
-            cList = this.controlTree.treeMap(treeFunc);
-            dList = this.dataTree.treeMap(treeFunc);
-
-            return util.mergeArray(cList, dList); //strip dupes and merge
```

Comment:
Looks like this was the only use of `mergeArray`, and  `_.uniq` does the same thing. Feel free to remove it from util.js.